### PR TITLE
Feature parameterize urls

### LINKED ISSUE #685: Bridge urls should be read from config file
In order to allow pointing atheneum to other runners/bibliotheca for offline an custom mumuki instalations, we nee to be able to customize them with a config config variable

### PR DESCRIPTION
**Pull request by flbulgarelli**
Fixes #685

### CODE REVIEW

**Comment by flbulgarelli** [comment]
Please review @aguspina . 

Please set the new env variables in all atheneum hosts before merging

File: app/models/concerns/with_status.rb [hunk 1]
```diff
@@ -10,10 +10,6 @@ def passed?
     status.passed?
   end
 
-  def errored?
```
Comment (aguspina) [inline app/models/concerns/with_status.rb]: if you delete this method... why is this
```ruby
if status.errored?
  []
else
  expectation_results.select { |it| it[:result] == :failed }
end
```

working?

Comment (aguspina) [inline app/models/concerns/with_status.rb]: Ok... moduler Errored implement this method... forget this comment

File: config/environments/development.rb [hunk 2]
```diff
@@ -43,8 +43,12 @@
   config.auth0_client_secret = ENV['MUMUKI_AUTH0_CLIENT_SECRET']
   config.auth0_domain = ENV['MUMUKI_AUTH0_DOMAIN']
 
-  config.offline_mode = ENV['OFFLINE_MODE']
-  config.domain_url = ENV['DOMAIN_URL'] || 'localmumuki.io:3000'
-  config.queueless_mode = ENV['QUEUELESS_MODE']
-  config.cookies_domain = ENV['COOKIES_DOMAIN'] || '.localmumuki.io'
+  config.offline_mode = ENV['MUMUKI_OFFLINE_MODE']
```
Comment (rodri042) [inline config/environments/development.rb]: Don't forget to update [this](https://github.com/mumuki/mumuki-offline/blob/236a3c273be2d36bf1b0869aebeffc5be88090ae/scripts/env.sh#L14) at merge

**Comment by flbulgarelli** [comment]
@aguspina @rodri042 could you review this PR again, please?

**Comment by aguspina** [comment]
Looks good to me!

**Comment by aguspina** [comment]
I'm going to merge this after setting env variables with escualo.rb

**Pull request merged**